spec/validations/mime_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe ZendeskAppsSupport::Validations::Mime do
```

Comment:
Nice work. Perhaps add a test (or two) to validate against a couple of block-listed files masked as safe ones?

Reply:
Thank you for the suggestion. I have added that.

I included a zip file that i renamed as something.html (in spec/fixtures)

MimeMagic still identifies it as a zip
![Screen Shot 2019-06-13 at 11 41 49 am](https://user-images.githubusercontent.com/17760485/59397569-62b12000-8dd0-11e9-8ae4-ee4b22f7845d.png)